### PR TITLE
[teleop] Joint b calibration

### PR DESCRIPTION
**Pull request by CameronTressler**

### CODE REVIEW

**Review by tabiosg — CHANGES_REQUESTED** [review]
Merge conflicts

**Comment by CameronTressler** [comment]
Oh thanks for fixing conflicts Bryan. Sorry for all the switching around with branches. Should be good now @tabiosg @cgiger00

**Review by tabiosg — APPROVED** [review]

**Review by bli76 — APPROVED** [review]

**Comment by CameronTressler** [comment]
Merge conflicts was just removing unnecessary toggle button styling

**Review by tabiosg — APPROVED** [review]

File: base_station/gui/src/components/LCMEcho.vue [hunk 1]
```diff
@@ -139,6 +138,7 @@
           {'topic': '/heater_cmd', 'type': 'Heater'},
           {'topic': '/heater_state_data', 'type': 'Heater'},
           {'topic': '/imu', 'type': 'IMU'},
+          {'topic': '/joint_b_calibration_data', 'type': 'JointBCalibration'},
```
Comment (cgiger00) [inline base_station/gui/src/components/LCMEcho.vue:141]: same comment as above

File: base_station/gui/src/components/SATask.vue [hunk 2]
```diff
@@ -219,12 +219,13 @@ export default {
...
         {'topic': '/sa_offset_pos', 'type': 'SAPosition'},
-        {'topic': '/arm_control_state_to_gui', 'type': 'ArmControlState'},
+        {'topic': '/arm_control_state', 'type': 'ArmControlState'},
         {'topic': '/heater_state_data', 'type': 'Heater'},
         {'topic': '/heater_auto_shutdown_data', 'type': 'HeaterAutoShutdown'},
         {'topic': '/pdb_data', 'type': 'PDBData'},
         {'topic': '/fuse_data', 'type': 'FuseData'},
-        {'topic': '/scoop_limit_switch_enable_cmd', 'type': 'ScoopLimitSwitchEnable'}
+        {'topic': '/scoop_limit_switch_enable_cmd', 'type': 'ScoopLimitSwitchEnable'},
+        {'topic': '/joint_b_calibration_data', 'type': 'JointBCalibration'}
```
Comment (cgiger00) [inline base_station/gui/src/components/SATask.vue:228]: same as above

File: base_station/gui/src/static/rover_msgs.json [hunk 3]
```diff
@@ -428,6 +428,12 @@
             "bearing"
         ]
     ],
+    "JointBCalibration": [
```
Comment (cgiger00) [inline base_station/gui/src/static/rover_msgs.json:431]: same as above

File: jetson/nucleo_bridge/README.md [hunk 4]
```diff
@@ -73,11 +73,6 @@ Message: [SAPosData.lcm](https://github.com/umrover/mrover-workspace/blob/master
 Publisher: jetson/nucleo_bridge \
 Subscriber: jetson/ra_kinematics + base_station/kineval_stencil + base_station/gui
 
-#### Wrist Turn Count \[Publisher\] "/wrist_turn_count"
-Message: [WristTurnCount.lcm](https://github.com/umrover/mrover-workspace/blob/master/rover_msgs/WristTurnCount.lcm) \
-Publisher: jetson/nucleo_bridge \
-Subscriber: base_station/gui + jetson/teleop
-
 #### Joint B Calibration Data \[Publisher\] "/joint_b_refresh_calibration_data"
 Message: [JointBCalibration.lcm](https://github.com/umrover/mrover-workspace/blob/master/rover_msgs/JointBCalibration.lcm) \
```
Comment (cgiger00) [inline jetson/nucleo_bridge/README.md:77]: same as above

**Review by cgiger00 — CHANGES_REQUESTED** [review]

**Review by cgiger00 — COMMENTED** [review]
has this ever been tested with joint B?

**Review by cgiger00 — APPROVED** [review]
^see above comment

**Comment by tabiosg** [comment]
No this hasn't been tested with joint-b

**Comment by tabiosg** [comment]
Ok it has been tested with joint-b now

**Review by bli76 — APPROVED** [review]

**Pull request merged**